controllers/storagecluster/uninstall_reconciler.go

```diff
@@ -297,20 +299,21 @@ func (r *StorageClusterReconciler) deleteResources(sc *ocsv1.StorageCluster) err
 	}
 
 	for _, obj := range objs {
-		err = obj.ensureDeleted(r, sc)
+		res, err := obj.ensureDeleted(r, sc)
 		if err != nil {
-			return err
+			res.RequeueAfter = time.Second * time.Duration(1)
+			return res, err
 		}
 	}
 
 	err = r.deleteNodeTaint(sc)
 	if err != nil {
-		return err
+		return reconcile.Result{RequeueAfter: time.Second * time.Duration(1)}, err
 	}
 
 	err = deleteKMSResources(r, sc)
 	if err != nil {
-		return err
+		return reconcile.Result{RequeueAfter: time.Second * time.Duration(1)}, err
```

Comment:
would be nice to use this result with 1 second timeout as a constant since its used in a lot of places.